iotedgedev/cli.py

```diff
@@ -61,14 +61,22 @@ def main(set_config, az_cli=None):
               required=False,
               help="Creates a new Azure IoT Edge Solution. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.")
 @click.argument("name", required=False)
-def solution(create, name):
+@click.option('--module',
```

Comment:
I would prefer to have a default module name.  Let's think of something that works well. I used filtermodule before, but can be anything.

Reply:
Shall we add a new environment entry like `DEFAULT_MODULE_NAME` which defaults to "filtermodule"?

Reply:
Sure

Reply:
I just realize that the environment variables loaded by .env file are scoped to a solution. Before the user runs the `iotedgedev solution --create` command, the .env file is not created yet. So the user will need to set the environment variable by himself. My current design is in https://github.com/Azure/iotedgedev/pull/194/files#diff-f7f95b5a237aedfa9c80a3ec57d525e7R66. Please let me know if you have better ideas. :)

Reply:
IC.  Yes, hardcode the default name, allow overrides via ENV or CMD line.